main.js

```diff
@@ -0,0 +1,20 @@
+const toggleButton = document.getElementById('toggleButton');
+const menu = document.getElementById('desktop-nav');
+const logo = document.getElementById('logo');
+
+const toggleMenu = () => {
+  if (menu.style.display === 'none') {
+    menu.style.display = 'block';
+    menu.style.height = '100vh';
+    logo.style.display = 'none';
+    toggleButton.childNodes[1].className = 'fas fa-times';
+    toggleButton.childNodes[1].style.color = 'grey';
+  } else {
+    logo.style.display = 'block';
+    menu.style.display = 'none';
+    toggleButton.childNodes[1].className = 'fas fa-bars';
+    toggleButton.childNodes[1].style.color = '#fff';
+  }
+};
+
+toggleButton.addEventListener('click', toggleMenu);
```

Comment:
It will be better if you will implement the closing nav bar function, after clicking on any link in it by displaying `none` 💯 👍